src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,566 @@
+import type p5 from 'p5'
+import {VisualizerDefault} from './VisualizerDefault'
+import type {VisualizerInterface} from '@/visualizers/VisualizerInterface'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+import type {SequenceInterface} from '../sequences/SequenceInterface'
+import * as math from 'mathjs'
+
+/** md
+
+# Primes and Sizes Visualizer
+
+[<img src="../../assets/img/primes-sizes/ring1.png" width="320" 
+style="margin-left: 1em; margin-right: 0.5em"
+/>](../assets/img/primes-sizes/ring1.png)
+
+The terms of the sequence are laid out in a grid, left-to-right and top
+to bottom.  All the primes appearing as divisors of the terms onscreen
+are assigned a colour.  Each term of the sequence is coloured as a blend
+of its component prime colours.  The term is drawn as a disk, whose brightness 
+varies according to the given function from outer rim to center.  It is
+possible to turn on `growth rings,' i.e. a modulus that the brightness is
+taken with respect to, so that tighter growth rings indicate faster growth.
+
+### Keyboard interaction
+
+- Press the **left arrow key** to show/hide the color of each prime factor.
+- Press the **right arrow key** to show/hide labels.
+- Press **[** to turn up brightness.
+- Press **]** to turn down brightness.
+
+### Parameters
+ **/
+
+const colorMap = new Map()
+
+class SeqColor extends VisualizerDefault implements VisualizerInterface {
+    name = 'Primes and Sizes'
+    n = 64
+    ringsBool = false
+    modulus = 25
+    formula = 'log(n^x)'
+
+    params = {
```

Comment:
still a bug with a095102, which seems to miss the top right disc when using the default settings

Reply:
fixed now

Reply:
OK, at this point A095102 is producing a nice rainbowy sequence of glyphs and all 64 appear with the default settings, so closing.